package/etc/conf.d/conflib/syslog/app-syslog-kaspersky_syslog.conf

```diff
@@ -25,8 +25,7 @@ block parser app-syslog-kaspersky_syslog() {
 };
 application app-syslog-kaspersky_syslog[sc4s-syslog-sdata] {
 	filter {
-        match('@23668' value('SDATA'))
-
+        match('event@23668' value('SDATA'))
```

Comment:
filter {                                                        
        match('@23668' value('.SDATA.*'))                   
    };

The filter should be updated as there is still a chance in main data there is event@23668888 etc.